folly/Portability.h

```diff
@@ -273,4 +273,10 @@ inline size_t malloc_usable_size(void* ptr) {
 }
 #endif
 
+#if defined(__x86_64__)
```

Comment:
This should be `#if defined(FOLLY_X64)`, which is what normally wraps the pause calls.